specification/components/examples/ResponseFullPermissions.yaml

```diff
@@ -1,4 +1,11 @@
-summary: Full permissions to medical record and services.
+summary: |
+  Patient has full permission to the current medical record, the current
+  medical record starts from the `effectiveDate` (1/11/2022). Access to
+  historical medical record is currently set to `summary` and limited to
```

Comment:
where it says "access to historical medical record is currently set to "summary" and limited to detailed" I think the "limited to detailed" sounds confusing, like it is summary or detailed? Maybe needs to use the words maximum requestable or similar